blockchain/sync_manager/src/sync.rs

```diff
@@ -233,6 +239,12 @@ impl<'a> Syncer<'a> {
         // see https://github.com/filecoin-project/lotus/blob/master/chain/sync.go#L611
         header.check_block_signature(header.miner_address())?;
 
+        // TODO: incomplete, still need to retrieve power in order to ensure ticket is the winner
+        let _slash = self.state_manager.miner_slashed(header.miner_address())?;
```

Comment:
Why do we check this? Can't correlate with spec

Reply:
https://github.com/filecoin-project/specs/blob/6ab401c0b92efb6420c6e198ec387cf56dc86057/validation.md

Under Semantical Validation: must be from a valid miner, i.e. has not been slashed

Reply:
That doesn't appear in the current spec

Reply:
The spec uses the term 'plausible miner' which to me is the equivalent to 'valid miner' the term given in the link I posted above which uses slashing as an example of what makes a 'valid miner', and in lotus, it's being used for their block validation. I am happy to remove but I want to make sure I provide relevant information as to why I included it.